apps/src/templates/rubrics/RubricContent.jsx

```diff
@@ -119,50 +119,56 @@ export default function RubricContent({
 
   return (
     <div className={style.rubricContent}>
-      {!!studentLevelInfo && <Heading2>{studentLevelInfo.name}</Heading2>}
-      <Heading5>
-        {i18n.lessonNumbered({
-          lessonNumber: lesson.position,
-          lessonName: lesson.name,
-        })}
-      </Heading5>
-      {!!studentLevelInfo && (
-        <div className={style.studentInfo}>
-          <div className={style.levelAndStudentDetails}>
-            {onLevelForEvaluation && (
-              <div className={style.studentMetadata}>
-                {studentLevelInfo.timeSpent && (
+      <div>
```

Comment:
This looks like a dramatic change, but I pretty much just added a div here and moved items inside it to make the grid work more nicely with each element.